src/person/person.go

```diff
@@ -71,3 +76,10 @@ func (p *Person) CanBeGE() bool {
 	}
 	return false
 }
+
+// IsBannerID returns true if id is a 9 digit number
+func IsBannerID(stringID string) bool {
+	clean := strings.Replace(stringID, "-", "", -1)
+	re := regexp.MustCompile("[0-9]{9}")
```

Comment:
We shoudl make sure this starts with 95.  I think it's actually safe as-is, but it would be a little safer just in case.